.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,52 @@
+name: Feature request
+description: Suggest an idea for this project
+title: "[Feature] <title>"
+labels: "feature"
```

```suggestion
labels: "Feature Proposal"
```

refer to https://github.com/euphony-io/euphony/pull/160
We're using `Feature Proposal` in euphony repository :)